server/routes.js

```diff
@@ -259,4 +271,22 @@ function fillDefaultSidebar(db) {
 				];
 				return context;
 			})
-}
\ No newline at end of file
+}
+
+function findFiles(db) {
+	return (context) =>
+		 fs.readdir( path.join(__dirname, "..", config.paths.upload) )
+		.then( (files) => {
+			context.files = files.map( (f) => {return {
+				path: path.join("/upload/", f),
+				delete: path.join("/static/", f), 
+				name: f.split("-").splice(1).join("-")
+			}})
+			context.files.unshift({
```

Comment:
I just realized what is going on here.  Please no lying about what files we have.

(See later comment on the pseudo-templating)